Loop/View Controllers/CarbAbsorptionViewController.swift

```diff
@@ -139,7 +139,7 @@ final class CarbAbsorptionViewController: LoopChartsTableViewController, Identif
         charts.updateEndDate(chartStartDate.addingTimeInterval(.hours(totalHours+1))) // When there is no data, this allows presenting current hour + 1
 
         let midnight = Calendar.current.startOfDay(for: Date())
-        let listStart = min(midnight, chartStartDate, Date(timeIntervalSinceNow: -deviceManager.carbStore.maximumAbsorptionTimeInterval))
+        let previousMidnight = midnight - .hours(24)
```

Comment:
Proposed change to see a week of data in the table (need this and the previous one):
```
        let previousMidnight = midnight - .hours(6*24)
```

Probably should also change `previousMidnight` to a new name - just in this one file. Perhaps `earliestTimeForCarbTable`.